src/js/views/patients/worklist/worklist-list.scss

```diff
@@ -119,3 +119,14 @@
 .worklist-list__search-helper {
   display: none;
 }
+
+.worklist-filter__no-owner {
```

Comment:
I'm having second thoughts about this class.  I don't want this button to break in the future if `button-primary` changes.  Let's make a new `button-filter-toggle` class that only extends `.button` in the buttons module and we can copy any other needed styles.  We can also go ahead and style the icon there like we do in button filter with the `.icon.fa-` so it's specific.  Then we're just toggling the `button-blue` class